pkg/ebpf/events_pipeline.go

```diff
@@ -275,7 +275,13 @@ func (t *Tracee) decodeEvents(ctx context.Context, sourceChan chan []byte) (<-ch
 			evt.Metadata = nil
 			// compute hashes using normalized times
 			evt.ThreadEntityId = utils.HashTaskID(eCtx.HostTid, uint64(evt.ThreadStartTime))
```

Comment:
Is is possible we've missed a normalization here in the above (passing `ThreadStartTime` and not `BootToEpochNS(...)`)?

Reply:
`evt.ThreadStartTime` is normalised in line 246.